_pages/partners.md

```diff
@@ -41,57 +41,45 @@ pyOpenSci is driven by a **vibrant community** of technical packaging experts, o
-</div>
 <div class="pyos-section purple">
 <div class="content" markdown="1">
+
+{% include feature_row id="partnerships" type="right" %}
+{% include feature_row id="partnership_benefits" type="right" %}
+
+</div>
+</div>
+
+{% include div_purple_bottom.html  %}
+
+<div class="pyos-section">
+<div class="content" markdown="1">
+
 ## Increase visibility for your communities' tools
```

Comment:
i hear you. Can you help me understand - is this comment specific to the title? OR just in general to the page. the title is great with your suggested text. i feel like making it longer would be hard. but we could definitely focus language on scientists as that is our intended audience.

Reply:
the comment is specific to the title, but as referenced in other comments, the page goes back and forth between talking "to" a specific audience as well as talking "about" a specific audience. I agree that in tutorials and educational content it makes sense to talk "to" the audience, but for a website it could go either way. my concern isn't which one we choose, just that whichever we choose, it's consistent across the website.